### DISCUSSION TITLE
Static site and the prefix strategy? How do you redirect?

### DISCUSSION THREAD

**Discussion by notflip** [nuxt-modules/i18n#1767] [Q&A]
I'm running a node SSR server at this moment, only because I'm using the prefix strategy, with the node preset `site.com/` gets redirected to `site.com/en`, so this all works fine.

However I can't seem to get this to work with a static site (which is what I prefer), so I'm wondering **has anyone managed to get a static site working with the redirect working? (I'm using nginx)**

**Comment by ineshbose** [comment]
This discussion seems to be a duplicate of https://github.com/nuxt-modules/i18n/discussions/1764. May I please ask you to use one discussion thread for an issue. Additionally, you could also join the official Nuxt discord (https://discord.com/invite/ps2h6QT) and ask questions in the `#i18n` channel.

**Reply by ineshbose** [reply]
Edit: I understand now that #1764 is regarding `index.html` in your build, while this is about redirecting using nginx - while the two seem closely related - so could I ask you to provide more description in just one discussion?
